packages/next/package.json

```diff
@@ -100,11 +100,13 @@
     ]
   },
   "dependencies": {
+    "@modelcontextprotocol/sdk": "1.15.1",
     "@next/env": "15.4.2-canary.10",
     "@swc/helpers": "0.5.15",
     "caniuse-lite": "^1.0.30001579",
     "postcss": "8.4.31",
-    "styled-jsx": "5.1.6"
+    "styled-jsx": "5.1.6",
+    "zod": "3.25.76"
```

Comment:
I think we use a compiled-in zod in some other places?

Reply:
Do we still need zod? zod is already the dep of @modelcontextprotocol/sdk